src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -214,6 +254,8 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
           {hasSavedEvents && (
             <Button
               onClick={downloadPersonalEventsCalendar}
+              // @debt downloadPersonalEventsCalendar seems to only download events for the currently selected day,
+              //  yet the UI here implies as though it will download all of my events
               customClass="NavBarSchedule__download-schedule-btn"
             >
               Download your schedule
```

Comment:
[debt] `downloadPersonalEventsCalendar` seems to only download events for the currently selected day, yet the UI here implies as though it will download all of my events